test/built-ins/TypedArray/prototype/Symbol.toStringTag/prop-desc.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.2.3.31
+description: >
+  "@@toStringTag" property of TypedArrayPrototype
+info: >
+  %TypedArray%.prototype.byteLength is an accessor property whose set accessor
+  function is undefined.
+
+  This property has the attributes { [[Enumerable]]: false, [[Configurable]]:
+  true }.
+includes: [propertyHelper.js, testTypedArray.js]
+features: [TypedArray]
+---*/
+
+var TypedArrayPrototype = TypedArray.prototype;
+var desc = Object.getOwnPropertyDescriptor(TypedArrayPrototype, 'byteLength');
```

Comment:
This file's description (and some of its body) concerns the unrelated`byteLength` property. Since this patch already has a file for the `byteLength`-related assertions, you can simply delete them from here.